cfg/auth.py

```diff
@@ -70,20 +71,19 @@ def dev_firebase_admin_auth():
 # get the api_key
 def get_api_key(tileserver):
     try:
-        with open('../cfg/config.cfg') as json_data_file:
+        with open(CONFIG_PATH) as json_data_file:
             data = json.load(json_data_file)
             api_key = data['imagery'][tileserver]
             return api_key
-    except:
-        print("Something is wrong with your API key."
-              "Do you even have an API key?")
-        sys.exit()
+    except Exception:
```

Comment:
This part's a little tricky because there's possibly 3 different errors here. The `IOError` from the file open. The `ValueError` from the JSON parsing. And the `KeyError` from the dictionary access. 

One thing you could do is at the top level open the configuration and store it as a dictionary constant. So, at around line 12 have something like: 

```
try: 
    with open(CONFIG_PATH) as f:
        config_data = f.read()
except IOError:
    print('Unable to load configuration file at {}, exiting'.format(CONFIG_PATH))
    sys.exit(1)

try:
    CONFIG = json.loads(config_data)
except ValueError:
    print("Unable to parse configuration file, likely because of malformed JSON, exiting")
    sys.exit(1)
```

Then at the various places where we load it, we can just reference it as `CONFIG`, like 

```
api_key = CONFIG['imagery'][tileserver]
```

That'd be a good step toward DRYing up the code a little bit.

Reply:
Thanks, @mathcass . Applied your suggestion with a slight modification to use `raise` so that we'd get the stack trace printed out (the exit code is still 1).